src/api/carts/use-carts.ts

```diff
@@ -0,0 +1,35 @@
+import { useInfiniteQuery } from '@tanstack/react-query';
+
+import { client } from '../common';
+import { cartKeys } from './query-keys';
+
+type Variables = {
+  limit?: number | string;
+  offset?: number | string;
+};
+
+const DEFAULT_LIMIT = 10;
+const DEFAULT_OFFSET = 0;
+
+const getCarts = async ({
+  limit = DEFAULT_LIMIT,
+  offset = DEFAULT_OFFSET,
+}: Variables) => {
+  const params = new URLSearchParams({
+    limit: limit.toString(),
+    offset: offset.toString(),
+  }).toString();
+  const { data } = await client.get(`carts?${params}`);
```

Comment:
Can we pass the `params` property in the Axios config object instead of concatenating the params in the URL, right?  I mean, at the end of the day it's the same, but we can go without the previous lines